packages/@aws-cdk/app-delivery/README.md

```diff
@@ -1,151 +1,132 @@
-## Continuous Integration / Continuous Delivery for CDK Applications
-This library includes a *CodePipeline* composite Action for deploying AWS CDK Applications.
+# App Delivery
```

Comment:
Have we made the previous method of building a CDK pipeline impossible now, or is that code still available? We're not really explaining how it works anymore at least.

I'm concerned because existing customers could be depending on this right now, and we'd break them with no real replacement. I would like this library to both export the CDK primitives to build this pipeline, as well as a the helper tool that builds a standard pipeline using those primitives from a config file, and explain both methods in the README.

The refactor is so big I can't really tell whether that is what's going on, but if not that's what I want :).

Reply:
This is basically a full replacement of the existing module. Start with the README file and judge if it offers the right building blocks. I’ll also see if we can expose more of the building blocks.

I’ll make sure to include a breaking change message in the commit. I don’t believe the current offering was of much value to anyone, especially given it did not support assets.